keep/api/arq_worker.py

```diff
@@ -84,7 +83,7 @@ def get_arq_worker(queue_name: str) -> Worker:
         "ARQ_KEEP_RESULT", cast=int, default=3600
     )  # duration to keep job results for
     expires = config(
-        "ARQ_EXPIRES", cast=int, default=3600
+        "ARQ_EXPIRES", cast=int, default=3600*1000
```

Comment:
I'll need @talboren opinion on that, it affects also other tasks not only AI

Reply:
That's fair, I updated the code and made it conditional (bigger only for ai-related tasks) @shahargl